### PR TITLE
added french translation

### PR DESCRIPTION
**Pull request by uncognic**
added french translation, everything translated manually by me.

### CODE REVIEW

File: README_fr_FR.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- Optimisation de la configuration de démarrage 
+- Planification optimisée des processus
+## Kit d'art
+Vous voulez créer votre propre fond d'écran Atlas? Peut-être jouer avec notre logo pour créers votre propre design? Nous avons ça accessible au public pour allumer de l'inspiration créative autour de la communauté[Regardez notre Kit d'art et créez quelque-chose d'époustouflant.](./img/brand-kit.zip)
+
+Nous avons aussi [une séctions dédiée dans l'onglet Communauté](https://github.com/Atlas-OS/Atlas/discussions/categories/community-artwork), pour que vous pourrez partager votre design avec d'autres artistes at créer de l'inspiration.
+
+## Disclaimer
+
+By downloading, modifying, or utilizing any of these images, you agree to [Microsoft's Terms.](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm) None of these images are pre-activated, you **must** use a genuine key.
```
Comment (sladewatkins) [inline README_fr_FR.md:79]: Should this line also be translated?

Comment (uncognic) [reply README_fr_FR.md:79]: I mean the chinese translation doesn't have it, so consistency?

**Review by sladewatkins — COMMENTED** [review]
Had a quick question about Line 74 in the translation: https://github.com/Atlas-OS/Atlas/pull/341#discussion_r1038616510_

File: README_fr_FR.md (hunk 1 above)
Comment (sladewatkins) [reply README_fr_FR.md:79]: > I mean the chinese translation doesn't have it, so consistency?

Just noticed that. Weird, but yeah, consistency.

**Comment by uncognic** [comment]
image
i responded

**Review by SyrmGIT — APPROVED** [review]

**Pull request merged**